### ISSUE TITLE
Compatibility with neural network: replacing with constant value instead of dropping the feature

### ISSUE THREAD

**Issue by stephanecollot** [aerdem4/lofo-importance#52]
Hi,

For neural network if you change the number of features, you need to change the input dimension and there for the number of neurones.
So, we could have an option like:
- leaving='drop' for current behaviour
- leaving='replace' for NN

What do you think?

**Comment by aerdem4** [comment]
You mean replacing with a constant value by "replace", right? Makes sense.

**Comment by stephanecollot** [comment]
yes, could be different leaving strategy. Good point for the batch normalization and the risk to divide by zero.

Side remark: with NN you will most probably use FLOFO instead of LOFO anyway, at least that fit my current use case, because I have too many features and training takes too long.
